content/experiments/quantum-dice.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Quantum Dice
+author: Russell Huffman
+description: A simple dice roller app using quantum computing to generate random numbers.
+image: https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
+to: /experiments/quantum-dice
+media:
+  - https://github.com/JRussellHuffman/quantum-dice/blob/master/assets/header-image.pngs
+  - https://medium.com/design-ibm/truly-quantum-dice-cfe372f4c586
```

Comment:
Cannot link to a medium post from media. This is intended for pictures and videos right now. Can you remove it, please.